backend/src/tasks/search-sync.ts

```diff
@@ -0,0 +1,54 @@
+import { Domain, connectToDatabase } from '../models';
+import { CommandOptions } from './ecs-client';
+import { In } from 'typeorm';
+import ESClient from './es-client';
+
+const MAX_RESULTS = 1000;
+
+export const handler = async (commandOptions: CommandOptions) => {
+  const { organizationId } = commandOptions;
+
+  console.log('Running searchSync');
+  await connectToDatabase();
+
+  const client = new ESClient();
+
+  const where = organizationId ? { organization: organizationId } : {};
+  let domains = await Domain.find({
+    where,
+    relations: ['services', 'organization', 'vulnerabilities']
+  });
+
+  domains = domains.filter((domain) => {
+    if (!domain.syncedAt) {
+      // Domain hasn't been synced before
+      return true;
+    }
+    const { syncedAt } = domain;
+    if (
+      domain.updatedAt > syncedAt ||
```

Comment:
Do you think we could change this logic to be on the DB query level?

Reply:
I tried doing that initially, but sequelize's logic is fairly limited for querying nested relations. I don't think this current approach would cause a prohibitively expensive performance hit.